scripts/wsrep_sst_rsync.sh

```diff
@@ -178,6 +178,21 @@ FILTER=(-f '- /lost+found'
 if [ "$WSREP_SST_OPT_ROLE" = "donor" ]
 then
 
+||||||| merged common ancestors
+cat << EOF > "$STUNNEL_CONF"
+key = $SSTKEY
+cert = $SSTCERT
+${CAFILE_OPT}
+foreground = yes
+pid = $STUNNEL_PID
+debug = warning
+client = yes
+connect = ${WSREP_SST_OPT_HOST_UNESCAPED}:${WSREP_SST_OPT_PORT}
+TIMEOUTclose = 0
+${VERIFY_OPT}
+EOF
+#connect = ${WSREP_SST_OPT_ADDR%/*}
```

Comment:
We reverted Codership's stunnel encryption in rsync by commit 998da9b5d. So this block can completely be removed.